db/seeds.rb

```diff
@@ -27,7 +27,11 @@
 CurriculumAffiliation.create(curriculum: fin, learning_unit: rails)
 CurriculumAffiliation.create(curriculum: fin, learning_unit: js) 
 CurriculumAffiliation.create(curriculum: fin, learning_unit: react)                                                         
-CurriculumAffiliation.create(curriculum: fin, learning_unit: nextjs) 
+CurriculumAffiliation.create(curriculum: fin, learning_unit: nextjs)
+
+Cycle.create(curriculum: fin, name: 'Ciclo 1', challenge: 'Crea una app en Rails', boilerplate_url: 'https://docs.docker.com/samples/rails/')
```

Comment:
Cycle model has a `learning_goals` attribute that has a not null restriction, and you didn't define any value, therefore this throw an error when running the seeds. 
The same applies for all cycles instances you create in this seed.